SDK/AppCenterData/Microsoft.AppCenter.Data.Android/packages.config

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="GoogleGson" version="2.8.5" targetFramework="monoandroid81" />
+  <package id="Microsoft.CSharp" version="4.3.0" targetFramework="monoandroid81" />
+  <package id="Newtonsoft.Json" version="10.0.1" targetFramework="monoandroid81" />
```

Comment:
How did we chose version to use since its not latest?
Also UWP SDK has a old version to avoid conflicts.